dashboard/src/actions/authActions.js

```diff
@@ -120,83 +64,15 @@ export const movePage = (toPage, navigate) => async (dispatch) => {
   navigate(toPage);
 };
 
-export const setUserLoggedInState = (value) => async (dispatch) => {
-  dispatch({
-    type: TYPES.KEEP_USER_LOGGED_IN,
-    payload: value,
-  });
-};
-
-export const registerUser =
-  (details, navigate) => async (dispatch, getState) => {
-    try {
-      dispatch({ type: TYPES.LOADING });
-      // empty the alerts
-      dispatch({
-        type: TYPES.USER_NOTION_ALERTS,
-        payload: [],
-      });
-      const endpoints = getState().apiEndpoint.endpoints;
-      const response = await API.post(endpoints?.api?.register, {
-        ...details,
-      });
-      if (response.status === 201) {
-        dispatch(showToast(SUCCESS, "Account created!", "Login to continue"));
-        navigate(APP_ROUTES.AUTH_LOGIN);
-      }
-      dispatch({ type: TYPES.COMPLETED });
-    } catch (error) {
-      const alerts = getState().userAuth.alerts;
-      let amsg = {};
-      document.querySelector(".signup-card").scrollTo(0, 0);
-      if (error?.response) {
-        amsg = error?.response?.data?.message;
-        dispatch(toggleSignupBtn(true));
-      } else {
-        amsg = error?.message;
-        dispatch({ type: TYPES.NETWORK_ERROR });
-      }
-      const alert = { title: amsg, key: uid() };
-      alerts.push(alert);
-      dispatch({
-        type: TYPES.USER_NOTION_ALERTS,
-        payload: alerts,
-      });
-      dispatch({ type: TYPES.COMPLETED });
-    }
-  };
-
-export const toggleSignupBtn = (isDisabled) => async (dispatch) => {
-  dispatch({
-    type: TYPES.SET_SIGNUP_BUTTON,
-    payload: isDisabled,
-  });
-};
-
-export const toggleLoginBtn = (isDisabled) => async (dispatch) => {
-  dispatch({
-    type: TYPES.SET_LOGIN_BUTTON,
-    payload: isDisabled,
-  });
-};
-
-export const getUserDetails = () => async (dispatch) => {
-  const loginDetails = {
-    isLoggedIn: Cookies.get("isLoggedIn"),
-    token: Cookies.get("token"),
-    username: Cookies.get("username"),
-  };
-  dispatch({
-    type: TYPES.SET_LOGIN_DETAILS,
-    payload: loginDetails,
-  });
```

Comment:
Please strip the dead cases out of `dashboard/src/reducers/authReducer.js` and the dead definitions out of `dashboard/src/actions/types.js`.